clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRecord.java

```diff
@@ -105,9 +114,31 @@ public TimestampType timestampType() {
         return timestampType;
     }
 
+    /**
+     * The checksum (CRC32) of the record.
+     */
+    public long checksum() {
+        return this.checksum;
+    }
+
+    /**
+     * The size of serialized, uncompressed key in bytes.
```

Comment:
Nitpick: should it be `The size of the serialized...`? There are other instances where the same comment applies.